code/modules/events/centcomm_order.dm

```diff
@@ -0,0 +1,83 @@
+//Refer to game/centcomm_orders.dm
+
+/datum/event/centcomm_order
+
+
+/datum/event/centcomm_order/start()
+	var/datum/centcomm_order/C = new
+	C.name = command_name()
+	if(prob(50))
+		C.must_be_in_crate = rand(0,1)
+
+	//Who is it paying to
+	var/department = pick("Cargo","Medical","Science")
+	var/list/chosen
+	C.acct = department_accounts[department]
+	switch(department)
+		if("Cargo") //Minerals
+			chosen = pick(
+				list(
+					"item" = /obj/item/stack/sheet/mineral/diamond,
+					"amount" = rand(5,50),
+					"value" = rand(400, 2500)
+					),
+				list(
+					"item" = /obj/item/stack/sheet/mineral/uranium,
+					"amount" = rand(5,50),
+					"value" = rand(200, 1500)
+					),
+				list(
+					"item" = /obj/item/stack/sheet/mineral/gold,
+					"amount" = rand(5,50),
+					"value" = rand(200, 1500)
+					),
+				list(
+					"item" = /obj/item/stack/sheet/mineral/silver,
+					"amount" = rand(5,50),
+					"value" = rand(200, 1500)
+					),
+				list(
+					"item" = /obj/item/stack/sheet/mineral/phazon,
+					"amount" = rand(1,10),
+					"value" = rand(400, 5000)
+					),
+				list(
+					"item" = /obj/item/stack/sheet/mineral/clown,
+					"amount" = rand(1,10),
+					"value" = rand(200, 3500)
+					)
+				)
+
+		if("Medical")
+			chosen = pick(
+				list(
+					"item" = /obj/item/weapon/gun/energy/gun/nuclear,
```

Comment:
Pretty sure this is an AEG, and science only gets modkits now

Reply:
Then science will have to get energy guns to upgrade from somewhere

Reply:
I just don't see it happening that science, cargo, and security all team up and sacrifice one of the few eguns on the station for, at most, 1250 credits.

Reply:
Can't cargo order a crate that has 2 eguns for 50 credits? I don't get what the issue is.